buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -120,6 +120,7 @@ class BuildPlugin implements Plugin<Project> {
                 println "  JDK Version           : ${gradleJavaVersionDetails}"
                 println "  JAVA_HOME             : ${gradleJavaHome}"
             }
+            println " Random Testing Seed    : ${project.testSeed}"
```

Comment:
Putting it in the RandomizedTestingPlugin makes sense but I was trying to stay away from that because I thought we'd move it out of ES's code and this feels a little ES specific. If not, I'm happy to move it there.